### PR TITLE
add backend for full_like, ones_like, zeros_like funcs

### PR DESCRIPTION
**Pull request by KsanaKozlova**

### CODE REVIEW

File: dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx [hunk 1]
```diff
@@ -112,6 +112,28 @@ cpdef dparray dpnp_full(result_shape, value_in, result_dtype):
     return result
 
 
+cpdef dparray dpnp_full_like(result_shape, value_in, result_dtype):
```
Comment (densmirn) [inline dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx:115]: This function is called nowhere.

**Review by densmirn — APPROVED** [review]

File: dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx (hunk 1 above)
Comment (shssf) [reply dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx:115]:
```
E     File "/home/vsts/work/1/s/tests/third_party/cupy/testing/helper.py", line 30, in _call_func
E       result = impl(self, *args, **kw)
E     File "/home/vsts/work/1/s/tests/third_party/cupy/creation_tests/test_basic.py", line 422, in test_zeros_like_reshape
E       return xp.zeros_like(a, order=order, shape=self.shape)
E     File "/home/vsts/work/1/s/dpnp/dpnp_iface_arraycreation.py", line 1205, in zeros_like
E       return dpnp_zeros_like(_shape, _dtype)
E   name 'dpnp_zeros_like' is not defined

```

Comment (shssf) [reply dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx:115]: @KsanaKozlova You forgot to put function names into following list
https://github.com/IntelPython/dpnp/blob/85aad6be56c4555855da62f0b21c978902448d24/dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx#L43-L56

Comment (samir-nasibli) [reply dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx:115]: @KsanaKozlova 
I know it is little bit annoying.
But I am saying it again and again: tests, tests and tests.

Try to run numpy tests for check:
https://github.com/IntelPython/dpnp#run-numpy-external-test

**Pull request merged**